recipe/meta.yaml

```diff
@@ -1,57 +1,297 @@
 {% set name = "pymedphys" %}
-{% set version = "0.7.2" %}
-{% set file_ext = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash_value = "52e68e40cb2245e46ba45e3c420f25056abc410448266e5e8f94fb79e1dc293c" %}
+{% set version = "0.8.5" %}
 
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
-  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
-  '{{ hash_type }}': '{{ hash_value }}'
+  - url: https://github.com/pymedphys/pymedphys/releases/download/v{{ version }}/v{{ version }}.tar.gz
+    sha256: 2cd3f6c09c861e9734f0048c8dbe309a482383470451840c793c1ddf824ede3d
 
 build:
-  number: 1
+  number: 0
   noarch: python
-  script: "python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
   entry_points:
     - pymedphys = pymedphys.cli.main:pymedphys_cli
+  script: "export SUBPACKAGE=pymedphys && yarn conda-forge"
+
 
 requirements:
-  build:
-    - python>=3.7
+  host:
+    - python >=3.7
     - pip
+    - yarn
   run:
-    - attrs
-    - keyring
-    - matplotlib
-    - notebook
-    - numpy>=1.12
-    - pandas
-    - Pillow
-    - psutil
-    - pydicom>=1.0
-    - pymssql
-    - python>=3.7
-    - python-dateutil
-    - scipy
-    - shapely
-    - xarray
+    - python >=3.7
+    - {{ pin_subpackage("pymedphys_analysis", exact=True) }}
+    - {{ pin_subpackage("pymedphys_base", exact=True) }}
+    - {{ pin_subpackage("pymedphys_coordsandscales", exact=True) }}
+    - {{ pin_subpackage("pymedphys_databases", exact=True) }}
+    - {{ pin_subpackage("pymedphys_dicom", exact=True) }}
+    - {{ pin_subpackage("pymedphys_electronfactors", exact=True) }}
+    - {{ pin_subpackage("pymedphys_fileformats", exact=True) }}
+    - {{ pin_subpackage("pymedphys_gamma", exact=True) }}
+    - {{ pin_subpackage("pymedphys_labs", exact=True) }}
+    - {{ pin_subpackage("pymedphys_logfiles", exact=True) }}
+    - {{ pin_subpackage("pymedphys_mudensity", exact=True) }}
+    - {{ pin_subpackage("pymedphys_utilities", exact=True) }}
+    - {{ pin_subpackage("pymedphys_xlwings", exact=True) }}
+
 
 test:
   imports:
     - pymedphys
+    - pymedphys.coordinates
+    - pymedphys.dicom
+    - pymedphys.electronfactors
+    - pymedphys.film
+    - pymedphys.gamma
+    - pymedphys.geometry
+    - pymedphys.logfile
+    - pymedphys.mephysto
+    - pymedphys.msq
+    - pymedphys.mudensity
+    - pymedphys.plt
+    - pymedphys.trf
+    - pymedphys.utilities
+
+
+outputs:
+  - name: pymedphys_analysis
+    build:
+      script: 
+        - "export SUBPACKAGE=pymedphys_analysis && yarn conda-forge"
```

Comment:
I'm not familiar with `yarn`. How does this work?

Reply:
It calls the following line:

https://github.com/pymedphys/pymedphys/blob/1d743f9db6457679c6ca803740c408683b805c0d/package.json#L13

Which then, depending on the environment variable `$SUBPACKAGE` then calls the respective line within the named package:

* https://github.com/pymedphys/pymedphys/blob/1d743f9db6457679c6ca803740c408683b805c0d/packages/pymedphys/package.json#L22
* https://github.com/pymedphys/pymedphys/blob/1d743f9db6457679c6ca803740c408683b805c0d/packages/pymedphys_sphinxtheme/package.json#L31
* https://github.com/pymedphys/pymedphys/blob/1d743f9db6457679c6ca803740c408683b805c0d/packages/pymedphys_analysis/package.json#L8